src/main/java/org/terasology/launcher/settings/LauncherSettingsValidator.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2016 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * This file is licensed under the MIT license.
+ * See the LICENSE file in project root for details.
+ */
+
+package org.terasology.launcher.settings;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.terasology.launcher.util.JavaHeapSize;
+
+import static org.terasology.launcher.settings.BaseLauncherSettings.PROPERTY_MAX_HEAP_SIZE;
+
+/**
+ * Provides methods to validate launcher settings
```

Comment:
Javadoc ends with fullstop.

Reply:
Sorry, I forgot to run the checkstyle. Updated it.